### PR TITLE
Adding UI for subscription Name for service bus topic binding.

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

File: Bindings/bindings.json [hunk 1]
```diff
@@ -455,6 +455,20 @@
                     ]
                 },
                 {
+                    "name": "subscriptionName",
+                    "value": "string",
+                    "defaultValue": "mysubscription",
+                    "required": true,
+                    "label": "Subscription name",
+                    "help": "This is the name of the subscription from which the message will be read.",
```
Comment (mathewc) [inline Bindings/bindings.json]: Recommend using the same or similar wording that the ServiceBusTriggerAttribute itself uses for these parameters

File: Bindings/bindings.json [hunk 2]
```diff
@@ -455,6 +455,20 @@
                     ]
                 },
                 {
+                    "name": "subscriptionName",
+                    "value": "string",
+                    "defaultValue": "mysubscription",
```
Comment (mathewc) [inline Bindings/bindings.json]: Use correct camel casing for variable names

Comment (mathewc) [inline Bindings/bindings.json]: What purpose does this default value serve? It will never be valid, right?

File: Bindings/bindings.json [hunk 3]
```diff
@@ -455,6 +455,20 @@
...
+                    "required": true,
+                    "label": "Subscription name",
+                    "help": "This is the name of the subscription from which the message will be read.",
+                    "validators": [
+                        {
+                            "expression": "^[0-9a-z][a-z0-9_.-]{1,48}[0-9a-z]$",
```
Comment (mathewc) [inline Bindings/bindings.json]: Where did you get this regex from - does it map to the actual naming rules for ServiceBus subscription names?

Comment (soninaren) [inline Bindings/bindings.json]: Got it from the old portal, It should allow uppercase characters though will update it.

File: Bindings/bindings.json (hunk 1 above)
Comment (soninaren) [inline Bindings/bindings.json]: what wording are you referring to?

Comment (mathewc) [inline Bindings/bindings.json]: https://github.com/Azure/azure-webjobs-sdk/blob/master/src/Microsoft.Azure.WebJobs.ServiceBus/ServiceBusTriggerAttribute.cs#L57

**Comment by soninaren** [comment]
updated the PR.

**Comment by mathewc** [comment]
:ship:

**Pull request merged**